include/zephyr/bluetooth/gatt.h

```diff
@@ -1772,9 +1777,27 @@ struct bt_gatt_read_params {
 			bool variable;
 		} multiple;
 		struct {
-			/** First requested handle number. */
+			/** @brief requested attribute handle number.
```

Comment:
Wondering if it might be nice to include start and end in the briefs of these?
```suggestion
			/** @brief requested start attribute handle number.
```
Also line 1792, then into something like `@brief requested end attribute handle number.`

Is just a nitpick though, and if you've thought about it and decided to keep it general in the briefs that is fine by me as well. Rest looks good to me.